cime_config/tests.py

```diff
@@ -305,6 +305,7 @@
             "ERS_Ld5.T62_oQU120.CMPASO-NYF",
             "ERS.f09_g16_g.MALISIA",
             "ERS_Ld5.TL319_oQU240wLI_ais8to30.MPAS_LISIO_JRA1p5.mpaso-ocn_glcshelf",
+            "ERS_Ld5.TL319_IcoswISC30E3r5_gis20.MPAS_LISIO_JRA1p5.mpaso-ocn_glc_tf_coupling",
```

Comment:
I think this should go in `e3sm_ocnice_stealth_features` for now.  It isn't appropriate for `e3sm_developer` in my view for now:
https://github.com/E3SM-Ocean-Discussion/E3SM/blob/f33a8621209837fed80fca74eb621e4e3b238898/cime_config/tests.py#L264-L270

Reply:
I had the same thought, but I was confused why we're not also considering `ERS_Ld5.TL319_oQU240wLI_ais8to30.MPAS_LISIO_JRA1p5.mpaso-ocn_glcshelf` a test for a stealth feature, and I thought these two ocn-glc features should be treated similarly.

Reply:
I think we decided that's not as stealthy a stealth feature.  We wanted to know if it's broken right away.  Stealth features don't get tested as rigorously.

Reply:
But that test could be moved to stealth as well until it's in a compset.